### PR TITLE
Add l20nchecker

### PR DESCRIPTION
**Pull request by ta2-1**
Adds l20ncheker for simple strings and exclude some checks for complex l20n entities.

### CODE REVIEW

**Comment by unho** [comment]
No tests?

**Comment by unho** [comment]
Other than the lack of tests which I would like to be addressed, looks ok.

File: translate/filters/test_checks.py [hunk 1]
```diff
@@ -1374,3 +1382,33 @@ def test_skip_checks_per_language_in_some_checkers():
...
+        }"""
+    )
+    unit = base.TranslationUnit(str1)
+    unit.target = str2
+
+    l20nchecker = checks.L20nChecker()
+    failures = l20nchecker.run_filters(unit)
+    assert 'variables' not in failures.keys()
+    assert 'brackets' not in failures.keys()
+    assert 'newlines' not in failures.keys()
```
Comment (unho) [inline translate/filters/test_checks.py]: Please also add a test to ensure that for a not complex unit these checks work.

File: translate/filters/test_checks.py [hunk 2]
```diff
@@ -1374,3 +1382,33 @@ def test_skip_checks_per_language_in_some_checkers():
     failures = stdchecker.run_filters(unit)
     # But it is not in StandardChecker.
     assert 'accelerators' in failures.keys()
+
+
+def test_skip_checks_for_l20n_complex_units():
+    """Test some checks are skipped for some languages in Mozilla checker."""
```
Comment (dwaynebailey) [inline translate/filters/test_checks.py]: l20n checker surely?

File: translate/filters/test_checks.py (hunk 1 above)
Comment (ta2-1) [inline translate/filters/test_checks.py]: I'm not sure I follow.
`test_variables_l20n` work with `variables` check. And the rest of checks are the same as in `MozillaChecker`.

Comment (ta2-1) [inline translate/filters/test_checks.py]: addressed

**Comment by unho** [comment]
lgtm

**Pull request merged**